centos_install_requirements.sh

```diff
@@ -7,7 +7,7 @@ source lib/logging.sh
 source lib/common.sh
 
 sudo dnf install -y libselinux-utils
-if selinuxenabled ; then
+if getenforce | grep Enforcing; then
```

Comment:
Wouldn't be better to check  SELinux mode explicitly against permissive?
```
if  !  getenforce | grep  Permissive; then
// do the settings
fi
```

Reply:
Only when getenforce  returns Enforcing, the settings are needed. when getenforce returns Disabled. the settings are not needed.